docs/software/devtools/debug/linaro.md

```diff
@@ -0,0 +1,294 @@
+
+## Quickstart guide
+
+The name of the uenv image is `linaro-forge`, and the available versions on a
+cluster can be determined using the `uenv image find` command, for example:
+```
+> uenv --version
+7.0.0
+
+> uenv image find linaro-forge
+uenv                    arch   system  id                size(MB)  date
+linaro-forge/24.1.1:v1  gh200  daint   e0e79f5c3e6a8ee0  365       2025-02-12
+
+> uenv image pull linaro-forge/24.1.1:v1
+pulling e0e79f5c3e6a8ee0 100.00% --- 365/365 (0.00 MB/s)
+```
+
+This uenv is configured to be mounted in the `/user-tools` path so that
+they can be used alongside application and development uenv mounted at
+`user-environment` (sidecar mode):
+
+=== "sidecar"
+
+    When using alongside another uenv, start a uenv session with both uenv as
+    follows, to mount the images at the respective `/user-environment` and
+    `/user-tools` locations:
+
+    ```bash
+    > uenv start prgenv-gnu/24.11:v1,linaro-forge/24.1.1:v1 --view=prgenv-gnu:default
+    > source /user-tools/activate
+
+    # test that everything has been mounted correctly
+    # (will give warnings if there are problems)
+    > uenv status
+
+    # check that ddt is in the path
+    > ddt --version
+    Linaro DDT Part of Linaro Forge.
+    Copyright (c) 2023-2024 Linaro Limited. All rights reserved.
+    Version: 24.1.1
+    ```
+
+=== "standalone"
+
+
+    ```bash
+    > uenv start linaro-forge/24.1.1:v1:/user-tools
+    > source /user-tools/activate
+
+    # check that ddt is in the path
+    ddt --version
+    Linaro DDT Part of Linaro Forge.
+    Copyright (c) 2023-2024 Linaro Limited. All rights reserved.
+    Version: 24.1.1
+    ```
+
+When using the uenv with no other environment mounted (standalone mode), the
+image will automatically be mounted in the `/user-tools` mount point. The
+`/user-tools/activate` script will make the forge executables available in your
+environment path.
+
+[//]: <> (The `/user-tools/activate` script will make the forge executables available)
+[//]: <> (in your environment, and **must be run before** any other uenv view command)
+[//]: <> (otherwise the environment variables set from the other uenv will be)
+[//]: <> (overwritten.)
+
+## User guide
+
+In order to debug your code on Alps, you need to:
+
+1. pull the linaro-forge uenv on one of the target Alps vCluster
+- install the Linaro Forge client on your local system (desktop/laptop)
+- build an executable with debug flags
+- launch a job with the debugger on Alps
+- start debugging/profiling
+
+### Pull the linaro-forge uenv on Alps
+
+The first step is to download the latest version of linaro-forge that is
+available on the system(s). First, SSH into the target system, then use the
+`uenv image find` command to list the available versions on the system:
+
+```
+> uenv image find linaro-forge
+uenv                    arch   system  id                size(MB)  date
+linaro-forge/24.1.1:v1  gh200  daint   e0e79f5c3e6a8ee0  365       2025-02-12
+```
```

Comment:
This is step-by-step guide so it's helpful to have this here, but it's a repetition of the above. I think it makes even more sense to have a collapsed example box above, but maybe we can even remove the whole thing since it's repeated here?